hedwiki/score/HED_score_0.0.1.mediawiki

```diff
@@ -403,11 +425,14 @@ Reference:
             *** Distal-limb
         ** Sensors <nowiki>[lists all corresponding sensors (electrodes/channels in montage)]</nowiki>
             *** <nowiki># {takesValue}</nowiki>
-        ** Propagation <nowiki>[Used with Findings-attributes tags]</nowiki>
-        ** Multifocal <nowiki>[Used with Findings-attributes tags]</nowiki>
-        ** Sleep-abnormal-amplitude-asymmetry <nowiki>[Absence or consistenly marked amplitude asymmetry (greater than 50 precent) of a normal sleep graphoelement]</nowiki>
-            *** Location-abnormal-amplitude-asymmetry-LowerLeft <nowiki>[Amplitude lower on the left side.]</nowiki>
-            *** Location-abnormal-amplitude-asymmetry-LowerRight <nowiki>[Amplitude lower on the right side.]</nowiki>
+        ** Propagation <nowiki>[When propagation within the graphoelement is observed, first the location of the onset region is scored. Then, the location of the propagation can be noted.]</nowiki>
```

Comment:
Concerned about `Propagation` standing alone.

Reply:
Changed to Finding-propagation